newIDE/app/src/EffectsList/index.js

```diff
@@ -486,213 +688,184 @@ export default function EffectsList(props: Props) {
-                                                effectMetadata.parametersSchema
-                                              }
-                                              project={props.project}
-                                              resourceManagementProps={
-                                                props.resourceManagementProps
-                                              }
-                                              renderExtraDescriptionText={
-                                                showEffectParameterNames
-                                                  ? parameterName =>
-                                                      i18n._(
-                                                        t`Parameter name in events: \`${parameterName}\` `
-                                                      )
-                                                  : undefined
-                                              }
-                                            />
-                                          </React.Fragment>
-                                        ) : null}
-                                        <Spacer />
-                                      </Column>
-                                    </Line>
-                                  )}
-                                </div>
-                              )
-                            }
-                          </DragSourceAndDropTarget>
-                        );
-                      }
-                    )}
+                {props.layerRenderingType !== '2d' && (
+                  <Column>
+                    <Column>
+                      <Line>
+                        <Text size="block-title">
+                          <Trans>3D effects</Trans>
+                        </Text>
+                      </Line>
+                    </Column>
+                    <Line>
+                      <Column noMargin expand>
+                        {mapFor(
+                          0,
+                          effectsContainer.getEffectsCount(),
+                          (i: number) => {
+                            const effect: gdEffect = effectsContainer.getEffectAt(
+                              i
+                            );
+                            const effectType = effect.getEffectType();
+                            const effectMetadata = getEnumeratedEffectMetadata(
+                              allEffectMetadata,
+                              effectType
+                            );
+
+                            const effectRef =
+                              justAddedEffectName === effect.getName()
+                                ? justAddedEffectElement
+                                : null;
+
+                            return !effectMetadata ||
+                              !effectMetadata.isMarkedAsOnlyWorkingFor2D ? (
+                              <DragSourceAndDropTarget
```

Comment:
I realize it's not a noop since the split is artificial. But still I would not give the impression it's doable so that we avoid having reports that dropping does nothing (although there's the drop indicator that shows it should work)